pyrene/src/components/TextArea/TextArea.tsx

```diff
@@ -89,10 +101,39 @@ const TextArea: React.FC<TextAreaProps> = ({
   onBlur,
   onChange,
   onFocus,
+  adaptToContent = false,
+  minRows = 1,
+  maxRows = 10,
 }: TextAreaProps) => {
 
+  const textAreaLineHeight = 16;
   const characterCount = maxLength - (value !== null ? value.length : 0);
   const characterLimitReached = characterCount < 0;
+  const textAreaRef = useRef<HTMLTextAreaElement>(null);
+  const [currentRows, setCurrentRows] = useState(minRows);
+  const [text, setText] = useState(value);
+
+  useEffect(() => {
+    if (adaptToContent && textAreaRef.current) {
+      // eslint-disable-next-line no-bitwise
```

Comment:
I guess this can also be removed this since we are no longer doing the bitwise operation